ee-9/deployment-transformer/src/main/java/org/wildfly/ee8to9/transformer/DeploymentTransformer.java

```diff
@@ -57,4 +62,9 @@ public void print(String format, Object... args) {
         });
         */
     }
+
+    private boolean isEnabled() {
+        final String value = WildFlySecurityManager.getPropertyPrivileged("org.wildfly.skip.jakarta.transformer", "");
```

Comment:
Please stick the word 'unsupported' in this property name.

How this stuff works can change at any time, and since the purpose of this AIUI is to ease use of WF Preview by outside projects, any projects that use this should understand that this isn't a reliable API.

Reply:
Done